README.md

```diff
@@ -1,14 +1,35 @@
-# ilp-plugin-payment-channel-framework [![npm][npm-image]][npm-url] [![circle][circle-image]][circle-url] [![codecov][codecov-image]][codecov-url]
+# ILP Plugin Payment Channel Framework [![npm][npm-image]][npm-url] [![circle][circle-image]][circle-url] [![codecov][codecov-image]][codecov-url]
 
-[npm-image]: https://img.shields.io/npm/v/ilp-plugin-virtual.svg?style=flat
-[npm-url]: https://npmjs.org/package/ilp-plugin-virtual
-[circle-image]: https://circleci.com/gh/interledgerjs/ilp-plugin-virtual.svg?style=shield
-[circle-url]: https://circleci.com/gh/interledgerjs/ilp-plugin-virtual
-[codecov-image]: https://codecov.io/gh/interledgerjs/ilp-plugin-virtual/branch/master/graph/badge.svg
-[codecov-url]: https://codecov.io/gh/interledgerjs/ilp-plugin-virtual
+[npm-image]: https://img.shields.io/npm/v/ilp-plugin-payment-channel-framework.svg?style=flat
+[npm-url]: https://npmjs.org/package/ilp-plugin-payment-channel-framework
+[circle-image]: https://circleci.com/gh/interledgerjs/ilp-plugin-payment-channel-framework.svg?style=shield
+[circle-url]: https://circleci.com/gh/interledgerjs/ilp-plugin-payment-channel-framework
+[codecov-image]: https://codecov.io/gh/interledgerjs/ilp-plugin-payment-channel-framework/branch/master/graph/badge.svg
+[codecov-url]: https://codecov.io/gh/interledgerjs/ilp-plugin-payment-channel-framework
 
-> ILP virtual ledger plugin for directly transacting connectors, including a
-> framework for attaching on-ledger settlement mechanisms.
+> Implements a trustline ledger plugin that can be extended to settle automatically via a payment channel
+
+The ILP Plugin Payment Channel Framework implements a [trustline](https://github.com/interledger/rfcs/blob/master/0022-hashed-timelock-agreements/0022-hashed-timelock-agreements.md#trustlines) that can be extended with an automatic settlement mechanism such as a payment channel.
+
+The framework itself provides:
+
+- An [HTTP RPC protocol](https://github.com/interledger/rfcs/blob/master/0021-plugin-rpc-api/0021-plugin-rpc-api.md) for plugin communication
+- Peer balance tracking and limit enforcement
+- Transfer and message validation
+- Tracking transfers in a log
+
+This module exports a `makePaymentChannelPlugin` function that
+takes a [Payment Channel Module](#payment-channel-module-api), and returns a
+LedgerPlugin class. Because trustlines and payment channels require maintaining state,
+users of these plugins must provide a more sophisticated [backend](#backend-api-with-extensions-for-payment-channels)
```

Comment:
the regular store in the connector can still be provided; an advanced backend is only required if it has to be multi-process